apiserver/apiserver.go

```diff
@@ -825,6 +826,7 @@ func (srv *Server) endpoints() ([]apihttp.Endpoint, error) {
 		resourceAuthFunc,
 		resourceChangeAllowedFunc,
 		&resourceServiceGetter{ctxt: httpCtxt},
+		resourcesdownload.NewDownloader(logger, resourcesdownload.DefaultFileSystem()),
```

Comment:
todo: Make the logs easier to differentiate in the new package: `logger.Child("resourcesdownload")`